src/Search.hs

```diff
@@ -0,0 +1,57 @@
+module Search
+    ( searchBoards
+    ) where
+
+import Control.Monad (mzero)
+import Data.Maybe (fromMaybe)
+import Data.List as List
```

Comment:
It was coming from `Data.Text` and `Data.ByteString.Lazy`. I limited what both of those were importing.